kernel/init.c

```diff
@@ -165,12 +165,7 @@ void __weak z_early_memcpy(void *dst, const void *src, size_t n)
 __boot_func
 void z_bss_zero(void)
 {
-	if (IS_ENABLED(CONFIG_ARCH_POSIX)) {
-		/* native_posix gets its memory cleared on entry by
-		 * the host OS, and in any case the host clang/lld
-		 * doesn't emit the __bss_end symbol this code expects
-		 * to see
```

Comment:
Let's move comment to the place where we select `SKIP_BSS_CLEAR` in `config ARCH_POSIX`

Reply:
Done, thanks!